.github/workflows/CI.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
         toolchain: [1.63.0, stable, nightly]
-        zig: [0.10.1, master]
+        zig: [0.9.0, 0.10.1, master]
```

Comment:
Maybe just drop 0.9.0? It has some issues, for example: https://github.com/ziglang/zig/issues/12046

Reply:
I wanted to test 0.9 in at least one platform because I observed some bugs that only occur in 0.9. The exact failure is that a Zig-produced binary somehow failed to read command-line arguments (see rust-lang/rust#66547 for the context) even with the latest available glibc, so I guess other platforms may still succeed. Trying once again.

Reply:
It seems that 0.9.0 is too unstable to fulfill my needs, so this line has been now reverted.